net-libs/libssh/libssh-0.8.4.recipe

```diff
@@ -57,43 +58,58 @@ BUILD_PREREQUIRES="
 	cmd:cmake
 	cmd:gcc$secondaryArchSuffix
 	cmd:ld$secondaryArchSuffix
-	cmd:libtoolize
 	cmd:make
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+if [ -z "$secondaryArchSuffix" ]; then
+	ARCHITECTURES_doc="any"
```

Comment:
Do we really want to move the doc to a subpackage ?
x86_gcc2(h) users won't be able to build it, nor to install it from a repo. The only way to install that doc subpackage would be to copy it from a x86_64 system and install it manually on the x86_gcc2(h) system.

Reply:
It's an `any` package isn't it? Shouldn't our builders be able to reuse the result from x86_64?

And I moved it since it's 2MB in size, while the `_devel` is only 224KB

Reply:
As you can image, it would be possible, technically speaking, and with "some" work, to teach the build bots running on x86_gcc2h to detect a recipe which is broken on x86_gcc2 primary arch but has one or more _"any"_-arch subpackages and have them fetched from the x86_64 repo. But I believe this would be too much work for a very small benefit. So until x86_gcc2 gets deprecated we'll probably have to keep docs in the base package for those recipes which do not build on x86_gcc2 primary arch.
That said, we can still make exceptions and decide that a given doc subpackage won't be available for x86_gcc2(h).
I don't have a strong opinion about this.
BTW, "any"-arch packages or subpackages created on x86_gcc2 and those created on x86_64 live in separate pools, so creating a doc subpackage here wouldn't even save any disk space.
BTW, a few "any"-arch packages (such as ExifTool if I'm not mistaken) are not perfectly indentical (because they have subdirectoris with names that refer to the architecture where they got built) so if we wanted to save some disk space we would first have to check that they are interchangeable, i.e. that e.g. an ExifTool package built on x86_64 can be installed on x86_gcc2 and work.

Related (closed) issue: https://github.com/haikuports/haikuporter/issues/84

Reply:
> BTW, "any"-arch packages or subpackages created on x86_gcc2 and those created on x86_64 live in separate pools, so creating a doc subpackage here wouldn't even save any disk space.

@pulkomandy, @waddlesplash can we have them shared, please? We can have an `any` pool like most distros out there.

> BTW, a few "any"-arch packages (such as ExifTool if I'm not mistaken) are not perfectly indentical (because they have subdirectoris with names that refer to the architecture where they got built) so if we wanted to save some disk space we would first have to check that they are interchangeable, i.e. that e.g. an ExifTool package built on x86_64 can be installed on x86_gcc2 and work.

That's a bug in the recipe. An `any`-arch package means that it's independent of the underlying architecture, if the contents depends on the arch that's used to build it, it's not `any`-arch.

Reply:
Let's just omit the docs on gcc2 for now. I think some changes should be made to the Haiku infrastructure.